src/common/com/intellij/plugins/haxe/model/type/HaxeTypeCompatible.java

```diff
@@ -53,7 +66,28 @@ static private boolean isAssignableFunction(SpecificFunctionReference a, Specifi
   }
 
   static private boolean isAssignableType(@NotNull SpecificHaxeClassReference a, @NotNull SpecificHaxeClassReference b) {
-    if (a.isDynamic() || b.isDynamic()) return true;
+    if (a.isDynamic() || b.isDynamic()) {
+      return true;
```

Comment:
I was evolving all that logic. Initially I just returned true. Then I was providing some more logic and the integration tests are checking indirectly all the new logic.
So probably it is not good. But I think we can easily add tests that fail and fix them so not a big issue.
Provide a test example that fails and I will fix it.